### ISSUE TITLE
Document on how to run tests locally

### ISSUE THREAD

**Issue by junakowicz** [byteball/byteballcore#86]
Describe simple steps on how to run tests

**Comment by tonyofbyteball** [comment]
good point, added a line about `npm test` https://github.com/byteball/byteballcore/commit/21b9cf5da06b61f5248f297db47a1f8a7c0d4eb1
